src/ppx/string_util.cpp

```diff
@@ -68,5 +69,34 @@ std::optional<std::pair<std::string_view, std::string_view>> SplitInTwo(std::str
     return std::make_pair(firstSubstring, secondSubstring);
 }
 
+std::string WrapText(const std::string& s, size_t width, size_t indent = 0)
```

Comment:
Could you move the default value to the function declaration in the header file?